known-issues.yml

```diff
@@ -30,6 +30,15 @@ pattern-ZibHealthProfessionalReference:
 
 zib-AddressInformation:
   zib deviations:
+    Address.line.extension:streetName.value[x]:
```

Comment:
There are some additions that do not seem to be related to this branch (I don't know if this is a problem).
Note: the changes w.r.t. EpisodeOfCare/HeadCircumference/HeartRate are the consequence of changing these to the right alphabetical order.

Reply:
Likely due to merging main into this branch and not resolving the conflicts correctly... Or not sure actually. Strange that this is a change compared to the main branch.
@pieter-edelman-nictiz , do you think this is incorrect?

Reply:
Looks like a failed merge conflict, I'll see if I can fix it. That being said, please don't touch the order in a branch; it makes merge conflicts so much harder to resolve. If there's something wrong, it should be fixed upstream.